src/main/java/org/isf/patient/service/PatientIoOperations.java

```diff
@@ -36,19 +36,14 @@
 import org.springframework.transaction.annotation.Transactional;
 
 /**
- * ------------------------------------------
- * PatientIoOperations - dB operations for the patient entity
- * -----------------------------------------
- * modification history
- * 05/05/2005 - giacomo  - first beta version
- * 03/11/2006 - ross - added toString method. Gestione apici per
- * nome, cognome, citta', indirizzo e note
- * 11/08/2008 - alessandro - added father & mother's names
- * 26/08/2008 - claudio    - added birth date
- * modified age
- * 01/01/2009 - Fabrizio   - changed the calls to PAT_AGE fields to
- * return again an int type
- * 03/12/2009 - Alex       - added method for merge two patients history
+ * ------------------------------------------ PatientIoOperations - dB
```

Comment:
I was not going to mention all the seemingly unneeded formatting changes until I saw this change that made the modification comment block unreadable.

Reply:
yes @AndreiDodu, maybe you can revert changes on all comments

Reply:
done